### DISCUSSION TITLE
How to change the backend design?

### DISCUSSION THREAD

**Discussion by biapar** [directus/directus#9785] [Archive]
How to change the backend design to have a better UI?

**Comment by benhaynes** [comment]
You'll need to clarify this question. "Better" is subjective, but you can use the project settings to change brand and CSS overrides, and since it's open source you can fork the repo and create your own version.

**Reply by biapar** [reply]
Better I mean to use a different font, color, color button, etc...

**Reply by azrikahar** [reply]
@biapar sounds like the Custom CSS in Project settings perfectly fits your use case here! In case you're not sure where it is, here's a quick guide on where it is as of version `9.0.1`:

![chrome_s1rQ5DoKdH](https://user-images.githubusercontent.com/42867097/141715181-5fb4a093-bb28-496f-a912-0d09f6bd8420.png)

1. Go to Settings Module
2. Click on `Project Settings`
3. Add your custom styles in the `Custom CSS` field here
